js_src/rdm.js

```diff
@@ -14,27 +17,24 @@ angular.module('rdmApp', [])
    convertor.euid = '';
    convertor.error = '';
 
-   if (!convertor.uid.match(/^[0-9a-fA-F]{4}:[0-9a-fA-F]{8}$/)) {
-    convertor.error = 'Invalid UID';
-    return;
-   }
-   var tokens = convertor.uid.split(':');
-   if (tokens.length !== 2) {
-    convertor.error = 'Invalid UID';
+   var clean_uid = convertor.uid.replace(/[\.:\-\s]/g, '');
+   if (!clean_uid.match(/^[0-9a-fA-F]{12}$/)) {
+    convertor.error = convertor.invalid_input_message;
```

Comment:
I don't think so, keep it simple :).

We have regex validation in the form now so the field turns green when the UID is valid. That means you hardly ever get these messages anymore.